### PR TITLE
chore(aws): Code Formatting

### PR DESCRIPTION
**Pull request by bbernays**
#### Summary

Reformatted code:
1. Code in `_fetch.go` files has been moved into the file that contains the table definition
2. All relational tables have been converted from public functions to private ones

### CODE REVIEW

File: plugins/source/aws/resources/services/elasticache/global_replication_groups_fetch.go [hunk 1]
```diff
@@ -1,21 +0,0 @@
...
-
-import (
-	"context"
-
-	"github.com/aws/aws-sdk-go-v2/service/elasticache"
-	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
-	"github.com/cloudquery/plugin-sdk/schema"
-)
-
-func fetchElasticacheGlobalReplicationGroups(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
```
Comment (candiduslynx) [inline plugins/source/aws/resources/services/elasticache/global_replication_groups_fetch.go:11]: should be moved to global_replication_groups.go

**Review by candiduslynx — CHANGES_REQUESTED** [review]

File: plugins/source/aws/resources/services/autoscaling/groups_fetch.go [hunk 2]
```diff
@@ -1,182 +1,14 @@
...
-func getNotificationConfigurationByGroupName(name string, set []types.NotificationConfiguration) []types.NotificationConfiguration {
-	var response []types.NotificationConfiguration
-	for _, s := range set {
-		if *s.AutoScalingGroupName == name {
-			response = append(response, s)
-		}
-	}
-	return response
-}
-
```
[the remark this answers is not in the document]
Comment (bbernays) [reply plugins/source/aws/resources/services/autoscaling/groups_fetch.go:179]: This single function is used in 3 other files, so I am fine leaving it by itself

File: plugins/source/aws/resources/services/athena/work_groups_fetch.go [hunk 3]
```diff
@@ -1,185 +1,12 @@
...
-	dc, err := svc.GetNamedQuery(ctx, &athena.GetNamedQueryInput{
-		NamedQueryId: aws.String(d),
-	})
-	if err != nil {
-		return err
-	}
-	resource.Item = *dc.NamedQuery
-	return nil
-}
-
```
[the remark this answers is not in the document]
Comment (bbernays) [reply plugins/source/aws/resources/services/athena/work_groups_fetch.go:182]: Great catch

File: plugins/source/aws/resources/services/athena/data_catalogs_fetch.go [hunk 4]
```diff
@@ -1,125 +1,12 @@
 package athena
 
 import (
-	"context"
```
[the remark this answers is not in the document]
Comment (bbernays) [reply plugins/source/aws/resources/services/athena/data_catalogs_fetch.go:4]: great catch

File: plugins/source/aws/resources/services/athena/work_groups_fetch.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (candiduslynx) [reply plugins/source/aws/resources/services/athena/work_groups_fetch.go:182]: especially having to review 804 files for that =)

**Review by candiduslynx — APPROVED** [review]

File: plugins/source/aws/resources/services/glue/dev_endpoints_fetch.go [hunk 5]
```diff
@@ -1,60 +1,12 @@
...
-		return err
-	}
-	return resource.Set(c.Name, result.Tags)
-}
-
-// ====================================================================================================================
-//                                                  User Defined Helpers
-// ====================================================================================================================
-
 func devEndpointARN(cl *client.Client, name string) string {
```
Comment (disq) [inline plugins/source/aws/resources/services/glue/dev_endpoints_fetch.go:10]: rename to `arn.go`?

File: plugins/source/aws/resources/services/route53/hosted_zones_fetch.go [hunk 6]
```diff
@@ -1,137 +1,9 @@
...
-		res <- response.TrafficPolicyInstances
-		if aws.ToString(response.TrafficPolicyInstanceNameMarker) == "" {
-			break
-		}
-		config.TrafficPolicyInstanceNameMarker = response.TrafficPolicyInstanceNameMarker
-	}
-	return nil
-}
-
 func getRoute53tagsByResourceID(id string, set []types.ResourceTagSet) []types.Tag {
```
Comment (disq) [inline plugins/source/aws/resources/services/route53/hosted_zones_fetch.go:7]: rename to `tags.go`? Also `getRoute53tagsByResourceID` => `getTagsByResourceID`

File: plugins/source/aws/resources/services/codepipeline/pipelines_mock_test.go [hunk 7]
```diff
@@ -44,6 +44,7 @@ func buildPipelines(t *testing.T, ctrl *gomock.Controller) client.Services {
 	if err := faker.FakeObject(&tags); err != nil {
 		t.Fatal(err)
 	}
+	tags.NextToken = nil
```
Comment (hermanschaaf) [inline plugins/source/aws/resources/services/codepipeline/pipelines_mock_test.go:47]: Why did this line change?

Comment (bbernays) [reply plugins/source/aws/resources/services/codepipeline/pipelines_mock_test.go:47]: Because we weren't handling pagination correctly initially.

Comment (bbernays) [reply plugins/source/aws/resources/services/codepipeline/pipelines_mock_test.go:47]: They shouldn't be. I thought I had a separate (local) branch that I was keeping with these fixes... Looking into it now

**Review by hermanschaaf — APPROVED** [review]
Spot-checked, LGTM but I'm mostly going to rely on the compiler and tests passing for this one 😅

**Pull request merged**